src/Tizen.NUI/src/public/Events/PanGestureDetector.cs

```diff
@@ -341,6 +341,18 @@ public void SetMaximumTouchesRequired(uint maximum)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// This value is a maximum duration of motion can live on the pan gesture event queue.
```

Comment:
The grammar is a little strange.
How about changing the sentence like this? :
 -> Sets a maximum duration of motion that is able to live(?) on the pan gesture event queue.

Reply:
Thank you for your advice. Fix done.